BeeSwift/AppDelegate.swift

```diff
@@ -143,4 +142,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return goal.relativeLane.intValue < -1
         }).count
     }
+    
+    // MARK: - UNUserNotificationCenterDelegate
+    
+    func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
```

Comment:
This will clear out the pending notifications, but will it silence the incoming notification that triggered it? Based on [the docs](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter) it looks like we might want to call the `completionHander` with `UNNotificationPresentationOptionNone`.

Reply:
adjusted to call the completionHandler and with an empty list of options in the case of "unlock phone" reminders while the app is active

Reply:
Sorry for the holdup here - I didn't understand why you used `[]` instead of `UNNotificationPresentationOptionNone` until I tried it myself and Xcode told me to just use `[]` :)